packages/manager/modules/pci/src/projects/new/payment/controller.js

```diff
@@ -153,9 +154,26 @@ export default class PciProjectNewPaymentCtrl {
         });
     }
 
-    return challengePromise.then(() => {
-      return !this.model.challenge.error ? this.manageProjectCreation() : null;
-    });
+    if (
+      this.eligibility.isDefaultPaymentMethodChoiceRequired() &&
+      this.model.defaultPaymentMethod
+    ) {
+      this.globalLoading.setDefaultPaymentMehtod = true;
+
+      defaultPaymentMethodPromise = this.ovhPaymentMethod
+        .setPaymentMethodAsDefault(this.model.defaultPaymentMethod)
+        .finally(() => {
+          this.globalLoading.setDefaultPaymentMehtod = false;
+        });
+    }
+
+    return Promise.all([challengePromise, defaultPaymentMethodPromise]).then(
```

Comment:
No issue with `$scope` digest ?

Reply:
Not here. Because `manageProjectCreation` is called in `then` callback and this method is handled with `$q`